### PR TITLE
Hardik_Goyal - DeepLearning_Contest

### PR DESCRIPTION
**Pull request by Hardik-Goyal**
CNN Model Implemented

> No. of Filters for different stages = 32, 64, 96, 128, 128.

> To reduce the spatial dimensions, max pooling is used. 

> Batch Size =128 , Epochs = 50.

> Epoch 50/50: ( loss: 0.3158 - accuracy: 0.8816 - val_loss: 0.5871 - val_accuracy: 0.8141 ).

### CODE REVIEW

**Comment by aitikgupta** [comment]
Thanks for the submission! 👍🏼

**Comment by Hardik-Goyal** [comment]
> Thanks for the submission! 👍🏼

Thanks for hosting such an Insightful Session and having this Competition sir!

Just like Webkriti it's so much keen to apply the concepts learnt in a session competitively.
It solidified the concepts learnt, and also I had to go the extra mile to learn about Keras functions and their hyperparameters. 

I wasn't satisfied with just one thing that the session could have been a bit longer and not rushed. It would have helped in better understanding of the Model Loss vs Epochs and the Model Accuracy vs Epochs graphs.
Rest all was Awesome!

**Comment by aitikgupta** [comment]
We're glad you had an amazing time! 💯 
I agree with you on the session, we'll improve that from next time. 👍🏼

Till then, keep contributing to open-source projects :)
